agent_non_root_condition.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+KEYLIME_GROUP=tss
+KEYLIME_USER=keylime
+
+# creating group if it isn't already there
+if ! getent group $KEYLIME_GROUP >/dev/null; then
+    addgroup --system $KEYLIME_GROUP
+fi
+
+# creating keylime user if he isn't already there
+if ! getent passwd keylime >/dev/null; then
+    adduser --system --ingroup tss --shell /bin/false \
```

Comment:
I think `--ingroup tss` should be `--ingroup ${KEYLIME_GROUP}` right?

Reply:
my mistake... just pushed a fix for it